src/screens/Home.js

```diff
@@ -113,12 +123,17 @@ export default ({
             <SectionHeading text="Seen phrases:" />
           </View>
           <List
-            data={[{id: 1, name: '35 words and phrases'}]}
+            data={[
+              {
+                id: '###seen-phrases###',
+                name: `${seenPhrases?.length} words and phrases`,
+              },
+            ]}
             text={'Learn'}
             color="#06B6D4"
             iconType="material-community"
             iconName="arrow-right"
-            makeAction={() => {}}
+            makeAction={seenPhrases?.length ? openSeenPhrases : () => {}}
```

Comment:
Create a variable for the repeated `seenPhrases?.length`

Reply:
Thanks, I have changed it.